packages/utils/src/orders.ts

```diff
@@ -0,0 +1,5 @@
+import { Order, OrderStatus } from '@impr3siones/models';
+
+export const completeOrder = (order: Order): void => {
+  order.status = OrderStatus.Completed;
+};
```

Comment:
También es bastante inútil, teniendo ya la utilidad de leftPadding esta podría desaparecer puesto que solo servía para comprobar si los imports de este paquete 📦  funcionaban bien

Reply:
Se mantiene porque al sacar los pedidos de la base de datos no se usa el leftPadding en front